frontend/src/components/Sidebar.vue

```diff
@@ -4,6 +4,7 @@
       <router-link
         class="action"
         to="/files/"
+        @click.native="refresh"
```

Comment:
The `router-link` component already handles the click event, adding another click directive will cause unexpected behavior. Instead of using a component, you should have a new element and handle its click behavior.

Reply:
Would you please fix it and pull a request? I am a newbie, I just don't know where to put the element.

Reply:
I will add it to my issues backlog and fix it when possible.

Reply:
Thank you so much.